### DISCUSSION TITLE
What is the differance between "Register-PnPAzureADApp" and "Initialize-PnPPowerShellAuthentication" ?

### DISCUSSION THREAD

**Discussion by khha** [pnp/powershell#3593] [General] [answered]
I already have experience with "Register-PnPAzureADApp" to create a new Azure app registration with certificate and api permissions. But now I would like to provision a new Team in MS Teams and found [this blog](https://www.collabmagazine.com/provisioning-teams-in-microsoft-teams-using-the-pnp-provisioning-engine/) from "Paolo Pailorsi". In he's blog he is using "Initialize-PnPPowerShellAuthentication". 

What is the differance between "Register-PnPAzureADApp" and "Initialize-PnPPowerShellAuthentication" ?

**Comment by AndersRask** [comment, marked as answer]
It was renamed to Register-PnPAzureADApp a couple of years back so Register-PnPAzureADApp is the correct to use

**Comment by khha** [comment]
> Initialize-PnPPowerShellAuthentication

But there is still official documentation about the old command. Is the old command deprecated?

https://pnp.github.io/powershell/cmdlets/Initialize-PnPPowerShellAuthentication.html

**Reply by AndersRask** [reply]
I dont think its part of the navigation, so it is the same that I have noticed with other deprecated cmdlets: old pages can still be found by search but are removed from navigation.

You cant use the command in newer versions since it is removed. The Register- cmdlet is the one you should use.
